companion/albumart.ts

```diff
@@ -27,12 +25,12 @@ export default function updateAlbumArt(albumArtURI: string) {
 
 function getAlbumArt(albumArtURI: string) {
     console.log("getting Album Art", albumArtURI);
-    fetch(albumArtURI, {}).then((response: any) => {
+    fetch(albumArtURI, {}).then((response) => {
```

Comment:
`fetch` now has it's own types: https://github.com/SergioMorchon/fitbit-sdk-types/blob/master/companion/fetch.d.ts